portfolio/src/main/java/com/google/sps/servlets/InformationServlet.java

```diff
@@ -11,18 +11,16 @@
 public final class InformationServlet extends HttpServlet {
   class MyInformation {
     public String[] messages;
+    MyInformation(String[] messages) {
+         this.messages = messages;
+    }
   }
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    // Convert the server info to JSON
-    String[] info = new String[3];
-    info[0] = "I enjoy basketball, badmiton, pingpong, and running";
-    info[1] = "I will learn tennis and boxing this summer";
-    info[2] = "I am learning stakeboarding now";
-    MyInformation information = new MyInformation();
-    information.messages = info;
+    final MyInformation information = new MyInformation(new String[]{"I enjoy basketball, badmiton, pingpong, and running",
+                                            "Learning tennis and boxing this summer", 
+                                            "Larning stakeboarding now"});
```

Comment:
So this and the `Gson()` object can/should be stored as private members of `InformationServlet`. Does that make sense?

```java
public final class InformationServlet extends HttpServlet {
  class MyInformation {
    public String[] messages;
    MyInformation(String[] messages) {
         this.messages = messages;
    }
  }

  private final MyInformation information = new MyInformation (...);
  private final Gson gson = new Gson();

    @Override
  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
  


     // Store as a variable, or better yet, just call this method like
    // response.getWriter().println(gson.toJson(info.messages));

    String json = gson.toJson(info.messages);

    response.setContentType("application/json;");
    response.getWriter().println(json);

  }


```